stun_server/region/network.tf

```diff
@@ -47,7 +47,7 @@ resource "aws_lb" "main" {
 
 resource "aws_lb_listener" "stun" {
   load_balancer_arn = aws_lb.main.arn
-  port              = 3478
+  port              = 80
```

Comment:
Do this need a code change elsewere?
Is this already used in core?

Reply:
Yes, we need code change in core.

Reply:
Is a PR for that open?
if not place this PR in draft.